src/renderer/services/docker/container.ts

```diff
@@ -202,4 +214,17 @@ export abstract class Container {
         }
         return false;
     }
+
+    public async remove(): Promise<boolean> {
+        if (this.docker && this.docker.name) {
+            try {
+                runCmd(Command.removeContainer(this.docker.name));
+                logger.info(`Docker container ${this.docker.name} removed.`);
```

Comment:
delete volume?

Reply:
Maybe it shouldn't be coupled? In case you want to leave data, idk

Reply:
Well you cannot connect that data back anyways haha

Reply:
Ok, wrong assumption then xD Will do